components/organisms/ProfileTabBody.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Button,
+  CalendarSVG,
+  CogSVG,
+  EthTransparentSVG,
+  HeartSVG,
+  Modal,
+  Skeleton,
+} from "@ensdomains/thorin";
+import { ProfileRecordItem } from "../02-molecules";
+import { DatabaseIcon } from "../01-atoms";
+import { formatDate, formatHexAddress } from "@/lib/utils/formats";
+import { CoinInfo, DomainData } from "@/lib/domain-page";
+import { useAccount, useEnsName } from "wagmi";
+import { useState } from "react";
+import { EditResolverModalContent } from "./EditResolverModalContent";
+import { useRouter } from "next/router";
+import { excludeKeys } from "@/pages/domains/[name]";
+
+export interface ProfileTabProps {
+  domainData: DomainData | null;
```

Comment:
github redirects me to this definition of [`DomainData`](https://github.com/blockful-io/external-resolver-dapp/blob/a23432aebbc4e10ec69ac59b14b0b6fd0363cac0/lib/utils/types.ts#L10) which doesn't have a `resolver` or `expiryDate`